core/services/aws/bloodDonation/createBloodDonation.ts

```diff
@@ -41,22 +42,24 @@ async function createBloodDonationLambda(
     }
     const response = await bloodDonationService.createBloodDonation(
       bloodDonationAttributes,
-      new DynamoDbTableOperations<
-      DonationDTO,
-      DonationFields,
-      BloodDonationModel
-      >(new BloodDonationModel()),
+      new DynamoDbTableOperations<DonationDTO, DonationFields, BloodDonationModel>(
+        new BloodDonationModel()
+      ),
       new BloodDonationModel()
     )
-    return generateApiGatewayResponse({ message: response }, HTTP_CODES.OK)
+    return generateApiGatewayResponse(
+      {
+        success: true,
+        message: CREATE_DONATION_REQUEST_SUCCESS,
+        data: response
+      },
+      HTTP_CODES.CREATED
+    )
   } catch (error) {
     httpLogger.error(error)
-    const errorMessage =
-      error instanceof Error ? error.message : 'An unknown error occurred'
+    const errorMessage = error instanceof Error ? error.message : 'An unknown error occurred'
```

Comment:
`An unknown error occurred` used in several places so we can declare a constant and then reuse it.

Reply:
Updated in 3fa44168bcc385ad9b8720e5461b55b387db8d9c